Readme.md

```diff
@@ -56,7 +56,7 @@ There are three options to uninstall Aerial from your Mac.
 
 * Right-click on the Aerial screensaver in System Preferences and select `Delete "Aerial"`.
 * Or delete the file(s) directly. macOS can store screen savers in two locations. `/Library/Screen Savers` and `/Users/YOURUSERNAME/Library/Screen Savers`. Check both locations for a file called "Aerial.saver" and delete any copies you find.
-* If you installed Aerial using Brew Cask, then the following command in a Terminal window should remove the Brew Cask installed version. `brew cask uninstall aerial`
+* If you installed Aerial using Brew Cask, then the enter the following command in a Terminal window that should remove the Brew Cask installed version: `brew cask uninstall aerial`.
```

Comment:
Then the enter the ? 

let's simplify it to

* If you installed Aerial using Brew Cask, then enter the following command in a Terminal window to uninstall: `brew cask uninstall aerial`.